src/pages/public/signup/index.js

```diff
@@ -1,7 +1,83 @@
+// import React, { useEffect } from "react";
 import React from "react";
+import { useFormik } from "formik";
 
 const SignUp = () => {
-  return <div></div>;
-};
+  const formik = useFormik({
+    initialValues: {
+      fullname: "",
+      email: "",
+      password: ""
+    },
+    onSubmit: (values) => {
+      const userFormDetail = {
+        userName: values.fullname,
+        useremail: values.email,
+        userPassword: values.password
+      };
+      console.log(userFormDetail.userName);
+      console.log(values.email);
+      const userDetail = localStorage.getItem("form")
+        ? JSON.parse(localStorage.getItem("form"))
+        : {};
+      if (userDetail[values.email]){
+        alert("The Following email exist already");
+      } 
+      else {
+        userDetail[values.email] = userFormDetail;
+        alert("Congratulation you are logged In");
+      }
+      localStorage.setItem("form", JSON.stringify(userDetail));
+      window.location.href = "http://localhost:3000/auth/login";
```

Comment:
aren't these login/signup two different screens? we should better use react router for redirection